src/resspect/plugin_utils.py

```diff
@@ -138,3 +139,28 @@ def fetch_query_strategy_class(query_strategy_name: str) -> QueryStrategy:
     """
 
     return get_or_load_class(query_strategy_name, QUERY_STRATEGY_REGISTRY)
+
+
+def fetch_feature_extractor_class(feature_extractor_name: str) -> LightCurve:
+    """Fetch the feature extractor class from the registry.
+
+    Parameters
+    ----------
+    feature_extract_name : str
+        The name of the feature extractor class to retrieve. This should either be the
+    name of the class or the import specification for the class.
+
+    Returns
+    -------
+    LightCurve
+        The subclass of LightCurve.
+
+    Raises
+    ------
+    ValueError
+        If a built-in query strategy was requested, but not found in the registry.
+    ValueError
+        If no query strategy was specified in the runtime configuration.
+    """
+
+    return get_or_load_class(feature_extractor_name, FEATURE_EXTRACTOR_REGISTRY)
```

Comment:
A thought here, since most of your changes are from `[a-z].*` to `[A-Z].*` what if we took the name and tried to see if it would match with the first letter made uppercase. If that is needed to match, output a deprecation warning with fix instructions.

It might lower the stakes for a user running into this on an old script, while also clearly specifying what is now correct behavior.

Reply:
This would probably look like running get_or_load_class twice, once after catching `ValueError` and a log line in the appropriate place.

Reply:
If you want an even more scoped-down exception, you could only exempt strings that correspond to valid feature extractors today. e.g. 
* "Bazin" and "bazin" both load the BazinFeatureExtractor class
* "Foo" loads the hypothetical future FooFeatureExtractor
* "foo" is a ValueError

Reply:
Yeah, I like the idea of being a little more lenient here so that pre-existing scripts don't break. That seems like that right approach.